app/admin/users.rb

```diff
@@ -0,0 +1,46 @@
+ActiveAdmin.register User do
+  permit_params :email, :password, :password_confirmation, :name, :gender, :avatar
+
+  index do
+    selectable_column
+    id_column
+    column :provider
+    column :uid
+    column :email
+    column :name
+    column :gender
+    column :push_token
+    column :current_sign_in_at
+    column :sign_in_count
+    column :created_at
+    column :updated_at
+    actions
+  end
+
+  filter :provider
+  filter :uid
+  filter :email
+  filter :name
+  filter :gender
+  filter :current_sign_in_at
+  filter :sign_in_count
+  filter :created_at
+  filter :updated_at
+
+  form do |f|
+    f.inputs do
+      f.input :email
+      f.input :name
+      f.input :gender
+      f.input :password
+      f.input :password_confirmation
+      f.input :push_token
```

Comment:
An admin should not be able to update this attribute right? This is set automatically when the user sign in to the app.

Reply:
[Here](https://github.com/sebastiancaraballo/target/pull/21) is a description about this attribute. I agree with removing it.